connectors/migrations/20241211_fix_gdrive_parents.ts

```diff
@@ -146,7 +122,7 @@ async function migrate({
         limit: QUERY_BATCH_SIZE,
       });
 
-    for (const sheet of googleDriveSheets) {
+    loop: for (const sheet of googleDriveSheets) {
```

Comment:
nit: Honestly a bit wary about the loop, always a bit hard to read
i'd have a small function computeParents instead of the while, and have it return undefined in error cases continuing without a label (you could also reuse it on docs)
but it's a nit so feel free to ignore